tests/unit/test_charm.py

```diff
@@ -964,6 +964,55 @@ def test_when_orchestrator_available_event_then_configuration_is_installed(
             ]
         )
 
+    @patch("netifaces.ifaddresses")
+    def test_given_eth1_interface_is_available_and_unit_is_leader_when_lte_core_relation_joined_then_then_core_information_is_set_and_charm_is_active(  # noqa: E501
+        self,
+        patch_ip_address,
+    ):
+        self.harness.set_leader(True)
+        patch_ip_address.return_value = {2: [{"addr": "0.0.0.0"}]}
+        relation_id = self.harness.add_relation("lte-core", "srs-enb-ue-operator")
+        self.harness.add_relation_unit(relation_id, "srs-enb-ue-operator/0")
+        self.assertEqual(
+            self.harness.get_relation_data(relation_id, self.charm.app),
+            {"mme_ipv4_address": "0.0.0.0"},
+        )
+        self.assertEqual(
+            self.charm.unit.status,
+            ActiveStatus(),
+        )
```

Comment:
How about splitting this into 2 different tests?

Reply:
Fixed